baseplate/integration/wrapped_context.py

```diff
@@ -0,0 +1,21 @@
+import copy
+
+
+def merge_two_dicts(x, y):
+    """Given two dicts, merge them into a new dict as a shallow copy."""
+    z = x.copy()
+    z.update(y)
+    return z
+
+
+class WrappedRequestContext(object):
+    def __init__(self, context, trace=None):
+        self._context = context
+        self.__dict__.update(self._context.__dict__)
```

Comment:
Why copy over the dict? Doesn't this make the `__getattr__` do nothing essentially? I think doing just the getattr would be cleanest since it keeps the wrapper free of any stuff in upstream stuff (e.g. does pyramid's request have a `clone` that would shadow `WrappedRequestContext`'s native clone?)